app/controllers/solicitations_controller.rb

```diff
@@ -1,132 +1,97 @@
-class SolicitationsController < ApplicationController
-  include TerritoryFiltrable
+class SolicitationsController < PagesController
+  before_action :find_solicitation, except: [:create]
 
-  before_action :find_solicitation, only: [:show, :update_status, :update_badges, :prepare_diagnosis, :ban_facility]
-  before_action :authorize_index_solicitation, :set_category_content, :setup_territory_filters, :count_solicitations, only: [:index, :processed, :canceled]
-  before_action :authorize_update_solicitation, only: [:update_status]
+  layout 'solicitation_form', except: [:form_complete]
 
-  layout 'side_menu'
-
-  def index
-    @solicitations = ordered_solicitations.status_in_progress
-    @status = t('solicitations.header.index')
-  end
-
-  def processed
-    @solicitations = ordered_solicitations.status_processed
-    @status = t('solicitations.header.processed')
-    render :index
+  def create
+    sanitized_params = sanitize_params(solicitation_params).merge(retrieve_query_params)
+    @solicitation = SolicitationModification::Create.new(sanitized_params).call!
+    if @solicitation.persisted?
+      session[:solicitation_form_id] = @solicitation.id
```

Comment:
Je n'ai pas vu ou tu supprime la session une fois fini

Reply:
je la supprimais, et finalement après un test où je revenais en arrière et où ça me mettait une erreur que je trouvais peu user friendly (not found, etc), je me suis dit que ça coûtait pas grand chose de laisser la variable, qui ne durera que le temps de la session de la personne qui dépose une sollication, ça me semble acceptable